ViewLayer/DebugDisplay/ControlPage/ControlView/ControlView.cpp

```diff
@@ -3,12 +3,15 @@
 
 namespace
 {
-    const QString MPPT_ALIVE = "background-color:rgb(93, 234, 140); \
+    const QString DRIVER_CONTROL_ALIVE = "background-color:rgb(93, 234, 140); \
                               border-radius:8px; \
                               border: 1px solid white;";
-    const QString MPPT_DEAD = "background-color:rgb(147, 147, 147); \
+    const QString DRIVER_CONTROL_DEAD = "background-color:rgb(147, 147, 147); \
                              border-radius:8px; \
                              border: 1px solid white;";
+    const QString ORANGE = "color: orange;";
```

Comment:
Instead or ORANGE and GREY, lets rename it to something more general, just in case we ever change the color to like blue or something.